src/consensus/parlia/seal.rs

```diff
@@ -134,7 +134,8 @@ impl SealBlock {
         let mut attestation = VoteAttestation::new_with_vote_data(VoteData {
```

Comment:
pls add a `TODO: add BEP-590 changes in fermi hardfork later`. It changes some vote assembly and verification logic;